src/main/java/net/openhft/chronicle/core/threads/EventLoop.java

```diff
@@ -84,4 +84,13 @@ static boolean inEventLoop() {
      */
     @Override
     void close();
+
+    /**
+     * Returns {@code true} if current thread is the core thread of this event loop, or if it cannot be determined.
+     * If current thread is not the core event thread but any other, returns {@code false}.
+     * @return {@code true} unless current thread is not the core event thread
+     */
+    default boolean runsInsideCoreLoop() {
+        return true;
```

Comment:
I think default value should be false

Reply:
Imagine somebody has an alternate event loop and we broke their code by introducing this check - I think it should be true because of that